apps/frontend/src/hooks/useWalletConnect.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { WalletState } from '@sovryn/onboard-core';
+
+import { onboard } from '../lib/connector';
+
+export const useWalletConnect = () => {
+  const [wallets, setWallets] = useState<WalletState[]>([]);
+
+  const connectWallet = useCallback(() => {
+    onboard.connectWallet('injected');
+  }, []);
+  const disconnectWallet = useCallback(async () => {
+    await onboard.disconnectWallet(wallets[0].label);
+  }, [wallets]);
```

Comment:
disconnect all wallets
```suggestion
  const disconnectWallet = useCallback(async () => {
    await onboard.disconnectWallet();
  }, [wallets]);
```